tests/sentry/integrations/gitlab/test_webhook.py

```diff
@@ -273,72 +273,74 @@ def test_merge_event_update_pull_request(self):
         self.assert_pull_request(pull, author)
         self.assert_group_link(group, pull)
 
-    def test_update_repo_name(self):
```

Comment:
Instead of commenting you can use `@pytest.mark.xfail(reason="renaming breaks 3 level repositories")`

Reply:
TIL, thanks!